src/MSAL.PCL/LoggerCallbackHandler.cs

```diff
@@ -78,7 +56,7 @@ public static IMsalLogCallback Callback
             }
         }
 
-        internal static void ExecuteCallback(LogLevel level, string message)
+        internal static void ExecuteCallback(MsalLogger.LogLevel level, string message)
```

Comment:
as per the spec, developer should be allowed to set callback only once. I think that change is missing in this class.